src/components/Application/PageElements.tsx

```diff
@@ -86,13 +92,86 @@ const PageElements: React.FC<PageElementProps> = ({
   if (isReview)
     return (
       <Form>
-        {elements.map((element) => {
-          return <p>Review Element</p>
-        })}
+        <Segment.Group>
+          {elements.map(({ element, latestOwnedReviewResponse }) => {
+            const summaryViewProps: SummaryViewWrapperPropsNEW = {
+              element,
+              response: responsesByCode?.[element.code],
+              allResponses: responsesByCode,
+            }
+            return (
+              <Segment key={`question_${element.id}`}>
+                <Grid columns="equal">
+                  <Grid.Column floated="left">
+                    <SummaryViewWrapperNEW {...summaryViewProps} />
+                  </Grid.Column>
+                  <Grid.Column floated="right" textAlign="right">
+                    <ReviewButton reviewResponse={latestOwnedReviewResponse as ReviewResponse} />
+                  </Grid.Column>
+                </Grid>
+                <ReviewResponseComponent
+                  reviewResponse={latestOwnedReviewResponse as ReviewResponse}
+                  summaryViewProps={summaryViewProps}
+                />
+              </Segment>
+            )
+          })}
+        </Segment.Group>
       </Form>
     )
 
   return null
 }
 
+const ReviewResponseComponent: React.FC<{
+  reviewResponse: ReviewResponse
+  summaryViewProps: SummaryViewWrapperPropsNEW
+}> = ({ reviewResponse, summaryViewProps }) => {
+  const [toggleDecisionArea, setToggleDecisionArea] = useState(false)
+
+  if (!reviewResponse) return null
+  if (!reviewResponse?.decision) return null
+
+  return (
+    <>
+      <Grid columns="equal">
+        <Grid.Column floated="left">
+          {`${reviewResponse?.decision}${reviewResponse?.comment || ''}`}
+        </Grid.Column>
+        <Grid.Column floated="right" textAlign="right">
+          <Icon
+            name="pencil"
+            color="blue"
+            onClick={() => setToggleDecisionArea(!toggleDecisionArea)}
+          />
+        </Grid.Column>
+      </Grid>
+      <DecisionAreaNEW
+        reviewResponse={reviewResponse}
+        toggle={toggleDecisionArea}
+        summaryViewProps={summaryViewProps}
+      />
+    </>
+  )
+}
+
+const ReviewButton: React.FC<{ reviewResponse: ReviewResponse }> = ({ reviewResponse }) => {
+  const [updateReviewResponse] = useUpdateReviewResponseMutation()
+
+  if (!reviewResponse) return null
+  if (reviewResponse?.decision) return null
+
+  return (
+    <Button
+      content={strings.BUTTON_REVIEW_RESPONSE}
+      size="small"
+      onClick={() =>
+        updateReviewResponse({
+          variables: { id: reviewResponse.id, decision: ReviewResponseDecision.Approve },
```

Comment:
```suggestion
          variables: { id: reviewResponse.id, decision: reviewResponse.decision, comment: reviewResponse.comment },
```

Reply:
don't ned to update comment when approving all right ?